pkg/credentialprovider/aws/aws_credentials.go

```diff
@@ -36,6 +36,28 @@ const awsStandardDNSSuffix = "amazonaws.com"
 const awsChinaDNSSuffix = "amazonaws.com.cn"
 const registryURLTemplate = "*.dkr.ecr.%s.%s"
 
+// AWSRegions is the complete list of regions known to the AWS cloudprovider
+// and credentialprovider.
+var AWSRegions = [...]string{
```

Comment:
There's PR open to consolidate some of the AWS regions logic https://github.com/kubernetes/kubernetes/pull/70675. I recommend waiting for that PR to go in first :)

Reply:
:+1: